_contentTemplates/common/get-started.md

```diff
@@ -13,59 +13,58 @@ The latest version of Telerik UI for Blazor is `{{site.uiForBlazorLatestVersion}
 
-If you prefer to do the process yourself, follow the [Setup the Telerik Private NuGet Feed]({%slug installation/nuget%}) article to set it up manually in case you don't have it already.
+If you prefer to configure the NuGet package source manually, follow the steps in the [Setup the Telerik Private NuGet Feed]({%slug installation/nuget%}) article.
 
-Once you have added the Telerik NuGet feed, continue with this tutorial.
+Once you have added the Telerik NuGet feed, continue by [enabling the components in the project](#enable-the-components-in-the-project).
 #end
 
 
 #project-creation-part-1
- If you have one, go to the [Add the Telerik Blazor Components to an Existing Project](#step-2---add-the-telerik-blazor-components-to-an-existing-project) section below.
-
-There are three ways to create the project:
 
-* [Create a Project with the Telerik VS Extensions](#create-a-project-with-the-telerik-vs-extensions)
+* If you already have one, go to the [Add the Telerik Blazor Components to an Existing Project](#step-2---add-the-telerik-blazor-components-to-an-existing-project) section below.
 
-* [Create a Project with the CLI](#create-a-project-with-the-cli)
+* To create a new project, choose one of the following methods:
 
-* [Create a Project with Visual Studio](#create-a-project-with-visual-studio)
+  * [Create a Project with the Telerik VS Extensions](#create-a-project-with-the-telerik-vs-extensions).
+  * [Create a Project with Visual Studio](#create-a-project-with-visual-studio).
+  * [Create a Project with the CLI](#create-a-project-with-the-cli).
 
 ### Create a Project with the Telerik VS Extensions
 
-You can use the [Visual Studio Extensions]({%slug getting-started-vs-integration-overview%}) we provide to [create the project for you]({%slug getting-started-vs-integration-new-project%}), so that you can start using the Telerik components immediately.
+You can use the [Telerik Visual Studio Extensions]({%slug getting-started-vs-integration-overview%}) that will [create and automatically configure the project]({%slug getting-started-vs-integration-new-project%}) so that you can start using the components immediately.
 
-If you use VS Code, you can also use our [VS Code Extension to create a Telerik-enabled project]({%slug getting-started-vs-code-integration-overview%}).
+>tip If you prefer VS Code, you can use the [VS Code Extension]({%slug getting-started-vs-code-integration-overview%}) to create a Telerik-enabled project.
 
-The rest of this article will explain the manual steps if you want to have a better understanding of the underlying process.
+If you choose to create a project with the Telerik VS Extensions, you can jump directly to [Step 3 - Add a Telerik Component to a View](#step-3---add-a-telerik-component-to-a-view). The following sections in this article explain the manual steps to configure the project so that you can better understand the underlying process.
 
-### Create a Project with the CLI
```

Comment:
I want to keep the section so there is a heading for SEO and for visibility, and I do not see a reason for much more details because CLI users generally know what the CLI is and how to use it.

I want to keep the extensions as the first section because it is the easiest for people. In fact, if they use our extensions, they don't have to follow this article further at all, the extensions will do everything for them. Not sure if we should remove the link to Step 3

Reply:
I agree that keeping the extensions as the first step is a good approach. My suggestion about removing the "Create a Project with the CLI" sections comes from the desire to have a leaner article.